In C23, void foo() is equivalent to void foo(void). Therefore, functions
must explicitly declare their parameters.

Downstream Bug: https://bugs.gentoo.org/945237
---
 someblocks.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/someblocks.c b/someblocks.c
index f149896..91de9d1 100644
--- a/someblocks.c
+++ b/someblocks.c
@@ -34,7 +34,7 @@ void setupsignals();
 void sighandler(int signum);
 int getstatus(char *str, char *last);
 void statusloop();
-void termhandler();
+void termhandler(int signum);
 void pstdout();
 void psomebar();
 static void (*writestatus) () = psomebar;
@@ -178,12 +178,12 @@ void sighandler(int signum)
 	writestatus();
 }
 
-void termhandler()
+void termhandler(int signum)
 {
 	statusContinue = 0;
 }
 
-void sigpipehandler()
+void sigpipehandler(int signum)
 {
 	close(somebarFd);
 	somebarFd = -1;
-- 
2.47.1
